config/electron.config.js

```diff
@@ -26,9 +28,11 @@ module.exports = {
         channel: 'latest',
     }],
     files: [
-        './dist/**/*',
-        './desktop/*.js',
-        './src/libs/checkForUpdates.js',
-        './src/CONST/ENVIRONMENT.js',
```

Comment:
Since webpack is bundling `main` as well  - we no longer have to explicitly list anything we import from `./src` here